examples/live-agent/README.md

```diff
@@ -0,0 +1,60 @@
+# Live Agent Example
+
+## Install Dependencies
+
+```sh
+yarn install
+```
+
+## Import Example Project
+
+Import this [project with custom actions](example_project.vf) into your Voiceflow workspace.
+It includes custom `account_info`, `calendar` and `video` actions.
+
+## Configure Environment
+
+Follow [these instructions](https://developer.voiceflow.com/docs/step-3-deploy-assistant) to publish your Voiceflow Assistant.
+
+Copy the project ID from the Assistant Settings and write it to a `.env.local` file.
+
+```sh
+# replace `XXX` with your project ID key
+echo 'VF_PROJECT_ID=XXX` > .env.local
+```
+
+## Run Dev Server
+
+The demo app will be available locally at <http://127.0.0.1:3006>.
+
+```sh
+yarn dev:app
+```
+
+You will also need to run the WebSocket server located in `server/` to interact with Intercom's APIs.
+See the [README](server/README.md) for more information.
+
+```sh
+yarn dev:server
+```
+
+### Run Simultaneously
+
+For convenience you can run both the chat widget and the WebSocket server at the same time with this command.
+
+```sh
+yarn dev
```

Comment:
do we want to lift this as the main go-to command higher on the file? Only bc it requires less steps to run the project